### ISSUE TITLE
[FE] - Accessibility testing artifacts

### ISSUE THREAD

**Issue by brianseek** [department-of-veterans-affairs/va.gov-team#51715]
Review new documentation on accessibility testing requirements and coordinate changes of findings with the rest of the team.

https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/change-history#Changehistory-December1,2022

**Comment by wullaski** [comment]
## Use of color and color contrast

As of WCAG 2.2 Focus state does not meet requirements https://www.w3.org/TR/WCAG22/#focus-appearance

## Automated testing with axe by Deque

Testing pointed out bad aria-describedby values
https://github.com/department-of-veterans-affairs/va.gov-team/issues/51894

## Content zoom and reflow

LGTM

## Keyboard navigation

LGTM

## Additional Findings

**WAVE Spot Checks**

Wave Spot check pointed out flaw with functional links such as the language picker.

![Screen Shot 2023-01-18 at 4 17 54 PM](https://user-images.githubusercontent.com/2982977/213783190-73545f9c-3e43-4cff-9a9e-652bc5c77fd5.png)

Created issue to address:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/52360

**Code quality review**

TODO:

**Mouse only and touchscreen**

TODO:

**Screen Readers**

Voice Over Testing pointed out issue with the loading component from the design system

Created an issue for the design system team
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1457

It also pointed out an issue with the hint text for the DOB input it doesn't mention the day field which also requires two digits

Created an issue with the design system team
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1455

Voice commands

TODO:
